Actions/AL-Go-Helper.ps1

```diff
@@ -1355,9 +1356,23 @@ function CommitFromNewFolder {
     invoke-git add *
     $status = invoke-git -returnValue status --porcelain=v1
     if ($status) {
+        $title = $commitMessage
+
+        # Add commit message suffix if specified in settings
+        $settings = ReadSettings
+        if ($settings.commitMessageSuffix) {
```

Comment:
I would have expected this would be an input field, rather than a setting???

Reply:
Right... I think both could make sense. 

If you run "Update AL-GO System Files" on a schedule I'm not sure the input would make sense. In that scenario I think a setting like this is better. But you're right... In the non-scheduled case an input would be a better option since it would give you more flexibility. 

Adding support for using the input field would be a bit of a heavier lift though (I think). IMO it could make sense to start with a simpler approach and expand it later if needed. What do you think? 

Motivation for the PR is really that I'm tired of having to manually edit the 10+ PRs we get whenever there are AL-Go updates to BCApps.

Reply:
And I'm tired of adding the "Automation" label + enabling the auto-merge on every PR from AL-Go to BCApps.
How about a more structured setting, as in https://github.com/microsoft/AL-Go/pull/1037/files#diff-d6e7039af6c0c11ad1a23decab42e0c9d1faeb7756089f0ebf640a2694665014

Pros of the settings: can be used in a scheduled workflows
Pros of the workflow input: it's more transparent what the workflow will do

Reply:
That too.. 
Don't know about this particular structured setting though. Presumably you'd need one for every workflow that creates PRs (since it's specific to Update AL-Go System Files)? Couldn't we leverage the "settings hierarchy"? If you want a setting to only apply to "Update AL-Go System Files" you need to create the `Update AL-Go System Files.settings.json`

Then we could have a single setting like: 
```
"workflowCommitSettings" = [ordered] @{
    "commitMessageSuffix": "Fixes AB#Something"
    "PRLabels": @("Automation")
    "AutocompletePR": $true
    "Whatever else": ...
 }
```

Reply:
Updated this to a structured setting with more options. Let me know what you think.